docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app

```

Comment:
Do not mount the whole directory as a volume in the final version, it should be done during development only